v3/cypress/e2e/graph.spec.ts

```diff
@@ -98,3 +98,41 @@ context("Graph UI", () => {
     })
   })
 })
+
+context("Graph adornments", () => {
```

Comment:
Nice job including cypress tests at this stage! I suspect this section will get quite long, so I think it would be better to start an `adornments.spec.ts` file.